test/setup.js

```diff
@@ -21,6 +21,7 @@ const historicDataTo = subDays(endOfYesterday(), 200)
 const historicDataFrom = startOfDay(subDays(historicDataTo, 205))
 
 const ethAddress = '0x1f3df0b8390bb8e9e322972c5e75583e87608ec2'
+const transactionType = 'ALL'
```

Comment:
Not needed here

Reply:
You're right, I put it in the test for the function and removed it from setup.js